CMakeLists.txt

```diff
@@ -21,7 +21,9 @@ include(cmake/define_libwiredtiger.cmake)
 # being defined (in base.cmake) since it will prevent it having a default value.
 if(NOT WT_ARCH)
     # Defer to our hosts architecture as our target architecture.
-    if ("${CMAKE_HOST_SYSTEM_PROCESSOR}" MATCHES "^(x86_64|i686|i386|AMD64)$")
+    # PROCESSOR_ARCHITECTURE env variable could be unset on certain Windows systems,
+    # in that case mapping the architecture to x86.
```

Comment:
`mapping` -> `map`

Reply:
Fixed in [d714263](https://github.com/wiredtiger/wiredtiger/pull/8772/commits/d714263002ab9093a48742333c6484821cfc5d7e)